homeassistant/components/meteoclimatic/translations/nl.json

```diff
@@ -12,7 +12,7 @@
                 "data": {
                     "code": "Station code"
                 },
-                "description": "Voer de code van het meteoklimatologische station in (bv. ESCAT43000043206B)",
+                "description": "Voer de code van het meteoklimatologische station in (bv. ESCAT4300000043206B)",
```

Comment:
Translations are managed on the Lokalise translation platform, please revert this change.

<https://developers.home-assistant.io/docs/translations>

Reply:
Done.